### PR TITLE
VEN-863 / kuva ci pipeline config

### PR DESCRIPTION
**Pull request by tebbon**
## Description :sparkles:
New docker base image, and gitlab-ci configuration to enable the Kuva CI pipeline and deployment environment.
Deployment cofiguration is not yet complete, some staging parameters are missing, but this needs to be merged in to enable the build stages.

### Related :handshake:
**[VEN-863](https://helsinkisolutionoffice.atlassian.net/browse/VEN-863):** 

### Automated tests :gear:️
The pipeline is it's own automated test.

### Manual testing :construction_worker_man:
Docker build tested manually, other changes 🤷

### CODE REVIEW

File: .gitlab-ci.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    K8S_SECRET_MAIL_MAILGUN_API: "https://api.eu.mailgun.net/v3"
+    K8S_SECRET_DEFAULT_FROM_EMAIL: "venepaikkavaraukset@hel.ninja"
+    K8S_SECRET_SENTRY_DSN: "$GL_SENTRY_DNS"
+    K8S_SECRET_SENTRY_ENVIRONMENT: "test"
+
+    K8S_SECRET_VENE_UI_RETURN_URL: "https://venepaikat.hel.ninja/"
+    K8S_SECRET_VENE_PAYMENTS_BAMBORA_API_URL": "https://real-bambora-api-url/api",
+    K8S_SECRET_VENE_PAYMENTS_BAMBORA_API_KEY": "dummy-key",
+    K8S_SECRET_VENE_PAYMENTS_BAMBORA_API_SECRET": "dummy-secret",
+    K8S_SECRET_VENE_PAYMENTS_BAMBORA_PAYMENT_METHODS": [ "dummy-bank" ]
```
Comment (a-rmz) [inline .gitlab-ci.yaml:75]: Those variables have different values for staging env, some of them should be private (api keys)

**Review by a-rmz — COMMENTED** [review]
Can you pls add a newline at the end of the files? 😄

File: .gitlab-ci.yaml (hunk 1 above)
Comment (tebbon) [reply .gitlab-ci.yaml:75]: Yes, we must fix the config separately, but let's first try to get the pipeline running, so placeholders for some configs for now.

**Review by santtul — APPROVED** [review]
Looks good, great job! I think we just need to revise the env variables and the uwsgi question, but let's give this a go already.

File: .gitlab-ci.yaml (hunk 1 above)
Comment (a-rmz) [reply .gitlab-ci.yaml:75]: Alright! Let's keep it in mind for the future 👍

**Review by a-rmz — APPROVED** [review]

**Pull request merged**